src/chunk.c

```diff
@@ -1406,13 +1563,36 @@ chunk_find_all(Hyperspace *hs, List *dimension_vecs, on_chunk_stub_func on_chunk
 Chunk **
 ts_chunk_find_all(Hyperspace *hs, List *dimension_vecs, LOCKMODE lockmode, unsigned int *num_chunks)
 {
-	return chunk_find_all(hs, dimension_vecs, append_chunk, lockmode, num_chunks);
+	Chunk **chunks = chunk_find_all(hs, dimension_vecs, append_chunk, lockmode, num_chunks);
+
+#ifdef USE_ASSERT_CHECKING
```

Comment:
AFAIK, it depends on how you configured your PostgreSQL build (`--enable-cassert`), which is true for all our `Assert`s. Unless I am mistaken, CodeCov indicates that this path is executed. Are you seeing something else?

Reply:
I see in CodeCov that it doesn't show as executed, but it doesn't show opposite either. I confirm that it should be configured for majority of the tests. So I believe I am wrong. It would be nice to understand why it is not shown in CodeCov, but it is not part of this issue.